icu4c/source/i18n/number_modifiers.h

```diff
@@ -272,60 +272,82 @@ class U_I18N_API EmptyModifier : public Modifier, public UMemory {
     bool fStrong;
 };
 
+/** An adopting Modifier store that varies by signum but not plural form. */
+class U_I18N_API AdoptingSignumModifierStore : public UMemory {
+  public:
```

Comment:
optional nit: labels are not normally indented

Reply:
I think elsewhere in this file, we have 2 spaces for the labels, and 4 spaces for everything else